pkg/vm/vm.go

```diff
@@ -1156,6 +1167,7 @@ func (v *VM) execute(ctx *Context, op Instruction, parameter []byte) {
 	default:
 		panic(fmt.Sprintf("unknown opcode %s", op.String()))
 	}
+	return err
```

Comment:
Either you should just `return` here, or you should remove `err` name from function result definition (but I'm not sure it'll work with `defer` then.